tqdm/_version.py

```diff
@@ -15,18 +11,41 @@
 
 
 # auto -extra based on commit hash (if not tagged as release)
-if (_sh is not None) and (len(version_info) < 4):  # pragma: no cover
-    def commit_hash(*args):
-        try:
-            res = _sh('git', 'log', '-n', '1', '--oneline', *args,
-                      stderr=STDOUT).lstrip().split()[0]
-            return None if res.startswith('fatal') else res
-        except:
-            return None
-
-    cur_hash = commit_hash()
-    if cur_hash is not None:
-        last_release = commit_hash('v' + __version__)
-
-        if (last_release is None) or (cur_hash not in last_release):
-            __version__ += '-' + cur_hash
+scriptdir = os.path.dirname(__file__)
+gitdir = os.path.abspath(scriptdir + '/../.git')
+if os.path.isdir(gitdir):
+    extra = None
+    # Open config file to check if we are in tqdm project
+    with open(os.path.join(gitdir, 'config'), 'r') as fh_config:
+        if 'tqdm' in fh_config.read():
```

Comment:
@sandrotosi Yep I thought the same but after reading a bit about git internals, I didn't find a better way to check if we were looking at tqdm repository. Any suggestion? Maybe looking specifically for a specific entry in config file?